server/cache.go

```diff
@@ -645,5 +701,33 @@ func (c *clusterInfo) handleRegionHeartbeat(region *RegionInfo) error {
 		}
 	}
 
+	c.updateWriteStatus(region)
+
 	return nil
 }
+
+func (c *clusterInfo) updateWriteStatus(region *RegionInfo) {
+	var WrittenBytesPerSec uint64
+	v, isExist := c.writeStatistics.peek(region.GetId())
+	if isExist {
+		interval := time.Now().Sub(v.(RegionStat).LastUpdateTime).Seconds()
+		if interval < minHotRegionReportInterval {
+			return
+		}
+		WrittenBytesPerSec = uint64(float64(region.WrittenBytes) / interval)
+	} else {
+		WrittenBytesPerSec = uint64(float64(region.WrittenBytes) / float64(regionHeartBeatReportInterval))
+	}
+	region.WrittenBytes = WrittenBytesPerSec
+
+	// hotRegionThreshold is use to pick hot region
+	// suppose the number of the hot regions is writeStatLRUMaxLen
+	// and we use total written Bytes past storeHeartBeatReportInterval seconds to divide the number of hot regions
+	// divide 2 because the store reports data about two times than the region record write to rocksdb
```

Comment:
At first we write raft log, and then apply log, so the physical write flow for rocksdb is about 2 times than the write flow of raftkv.